day13/BOJ_12904_A와B/BOJ_12904_A와B_신영제.java

```diff
@@ -0,0 +1,36 @@
+package com.ssafy.study.homework13;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+/*
+ * (1) 그리디 알고리즘
+ * (2) T를 기준으로 제거해 간다.
+ */
+
+public class BOJ_12904_A와B {
+
+	public static void main(String[] args) throws IOException {
+		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+		String S = br.readLine();
+		String T = br.readLine();
+		
+		int Tlen = T.length();
+		
+		for(int i = 0; i < Tlen; i++) {
+			if (T.charAt(T.length() - 1) == 'A') {
+				T = T.substring(0, T.length() - 1);
+			} else if (T.charAt(T.length() - 1) == 'B') {
```

Comment:
```suggestion
                        } else {
```
A 또는 B로만 구성된 단어이기 때문에 else를 사용해도 좋을 것 같습니다.